pkg/collector/helm_collector.go

```diff
@@ -0,0 +1,80 @@
+package collector
+
+import (
+	"github.com/Azure/aks-periscope/pkg/interfaces"
+	"github.com/Azure/aks-periscope/pkg/utils"
+	"path/filepath"
+)
+
+// HelmCollector defines a Helm Collector struct
+type HelmCollector struct {
+	BaseCollector
+}
+
+var _ interfaces.Collector = &HelmCollector{}
+
+// NewHelmCollector is a constructor
+func NewHelmCollector(exporter interfaces.Exporter) *HelmCollector {
+	return &HelmCollector{
+		BaseCollector: BaseCollector{
+			collectorType: Helm,
+			exporter:      exporter,
+		},
+	}
+}
+
+// Collect implements the interface method
+func (collector *HelmCollector) Collect() error {
+	rootPath, err := utils.CreateCollectorDir(collector.GetName())
+	if err != nil {
+		return err
+	}
+
+	output, err := utils.RunCommandOnContainer("apk", "add", "curl", "openssl", "bash", "--no-cache")
```

Comment:
💡 🐛 I plan to add PR to your fork tonight, but see I noticed something interesting: 

`:=` is Short variable declaration, **but** if you see carefully, up until line 53 you don't do anything with the `output` var.

I can make small updates and send small PR to your fort by tonight. Thanks.